string type

# type="pose_goal"
inhus/PoseGoal pose_goal

# type="named_goal"
inhus/NamedGoal named_goal